ddtrace/contrib/rq/__init__.py

```diff
@@ -0,0 +1,245 @@
+"""
+The RQ integration will trace your jobs.
+
+Usage
+~~~~~
+
+The rq integration is enabled automatically when using
+:ref:`ddtrace-run<ddtracerun>` or :ref:`patch_all()<patch_all>`.
+
+Or use :ref:`patch()<patch>` to manually enable the integration::
+
+    from ddtrace import patch
+    patch(rq=True)
+
+
+Worker Usage
+~~~~~~~~~~~~
+
+``ddtrace-run`` can be used to easily trace your workers::
+
+    DD_RQ_WORKER_SERVICE=myworker ddtrace-run rq worker
```

Comment:
```suggestion
    DD_SERVICE=myworker ddtrace-run rq worker
```

Reply:
hmmm, I wonder if we should get rid of the worker separate service name and just keep as a single service name for the integration which defaults to `DD_SERVICE` or maybe `rq` if unset ?